leftwm/doc/leftwm.1

```diff
@@ -0,0 +1,260 @@
+Mod + Enter;Move selected window to the top of the stack in the current workspace
+Mod + Ctrl + (up or down);Switch between different layouts
+Mod + Shift + (left or right);Switch between different workspaces
+Mod + Shift + Enter;Open a terminal
+Mod + Ctrl + L;Lock the screen
+Mod + Shift + X;Exit LeftWM
+Mod + Shift + Q;Close the current window
+Mod + Shift + R;Reload LeftWM and its config
+Mod + p;Use dmenu to start application
+.TE
+.PP
+.B Note:
+Although we encourage you to use Alacritty, LeftWM will set your default terminal to the first terminal it finds in this list (in the order presented):
+.IP \[bu] 2
+Alacritty
+.IP \[bu] 2
+Termite
+.IP \[bu] 2
+Kitty
+.IP \[bu] 2
+URXVT
+.IP \[bu] 2
+RXVT
+.IP \[bu] 2
+ST
+.IP \[bu] 2
+ROXTerm
+.IP \[bu] 2
+Eterm
+.IP \[bu] 2
+XTerm
+.IP \[bu] 2
+Terminator
+.IP \[bu] 2
+Terminology
+.IP \[bu] 2
+Gnome Terminal
+.IP \[bu] 2
+XFCE4 Terminal
+.IP \[bu] 2
+Konsole
+.IP \[bu] 2
+UXTerm
+.IP \[bu] 2
+Guake
+
+.IP "About Keybind Modifiers"
+Note: All entries require a modifier, even if blank: modifier = []
+see your
+.I
+config.toml
+file for more information.
+
+.SS Floating Windows
+.PP
+You can optionally switch between tiling or floating mode for any
+window.
+.PP
+.TS
+tab(;);
+l l.
+
+Keybinding;Description
+_
+
+Mod + MouseDrag ; Switch a tiled window to floating mode
+Mod + RightMouseDrag ; Resize a window
+Drag window onto a tile ; Switch a floating window to tiling mode
+.TE
+.SS FloatingToTile
+This behaviour snaps the currenly focused window into the workspace below.
+.SS TileToFloating
+This behaviour switches the currently focused window to floating mode when it's tiled.
+.SS ToggleFloating
+This behaviour switches the currenly focused window between floating and tiled mode.
+
+.\" Configuration section\"
+.SH CONFIGURATION
+.B IMPORTANT:
+You will need to reload in order to apply any changes you write to
+.I config.toml
+
+.SS Modkey
+.PP
+The modkey is the most important setting.
+It is used by many other settings and controls how key bindings work:
+.PP
+Default: \f[C]modkey = \[dq]Mod4\[dq]\f[R] (windows key)
+.SS Mousekey
+.PP
+The mousekey is similarly quite important.
+This value can be used to determine which key, when held, can assist a
+mouse drag in resizing or moving a floating window or making a window
+float or tile.
+.PP
+Default: \f[C]mousekey = \[dq]Mod4\[dq]\f[R] (windows key)
+.SS Focus Behaviour
+.PP
+LeftWM has 3 focusing behaviours (Sloppy, ClickTo, and Driven) and
+one option (focus_new_windows), which alter the way focus is handled.
+These encompass 4 different patterns:
+.IP "1." 3
+Sloppy Focus.
+Focus follows the mouse, hovering over a window brings it to focus.
+.IP "2." 3
+Click-to-Focus.
+Focus follows the mouse, but only clicks change focus.
+.IP "3." 3
+Driven Focus.
+Focus disregards the mouse, only keyboard actions drive the focus.
+.IP "4." 3
+Event Focus.
+Focuses when requested by the window/new windows.
```

Comment:
I don't think "Event" is a thing (anymore)?
https://github.com/leftwm/leftwm/blob/589dc71d2a98500c83c2a94210d4d2e80773bc7b/leftwm-core/src/models/focus_manager.rs#L10-L14